### PR TITLE
 Comparing configs ( latest commit to config branch Sep 14) with the current working Hpc speed debug branch

### PR DESCRIPTION
**Pull request by TJTapajyoti**

### CODE REVIEW

**Comment by TJTapajyoti** [comment]
@akey7 This pull request shows the differences properly. 

I think the issue was the pull request being directed properly. This is following the different direction of comparison.

**Comment by TJTapajyoti** [comment]
@rjhanes Are the changes for compute_locations.py doing something entirely new? DO we need to regenerate locations_computed.csv with this file? Currently compute_locations = False in my runs.

**Comment by rjhanes** [comment]
@TJTapajyoti Yes, compute_locations now has:

- the filtering built-in so that the locations file and the turbine data file will always be consistent
- automated generation of the step_costs file
- capacity expansion calculations that add future power plant locations and turbine data

The workflow remains the same, however: If you're executing many runs on the same input dataset, then only on the first run do the locations, turbine data, and step_costs files need to be generated. It takes a few seconds to complete that preprocessing with the national dataset.

So, basically, when `configs` is merged into `develop`, I'd do one national-scale run just to get the updated input files (including routes), and then filter those files as needed for subsequent runs.

**Comment by TJTapajyoti** [comment]
Closing PR as speed issue has been resolved.
